prov/efa/src/rxr/rxr.h

```diff
@@ -445,11 +445,14 @@ struct rxr_tx_entry {
 	uint16_t credit_allocated;
 
 	uint64_t total_len;
+	bool receipt_received;
```

Comment:
Because you have already introduce ```rxr_flags```, you do not need ```receipt_received```. you can introduce a new flag ```RXR_RECEIPT_RECEIVED``` and set it in ```rxr_flags```.

Reply:
Member removed. Resoving.